script-test/playbackstrategies/msestrategytest.js

```diff
@@ -792,13 +801,13 @@ require(
 
         it('should call plugins with video playback bitrate', function () {
```

Comment:
Maybe rename this test to make it clear this is the video and audio bitrate case?